docs/build.rst

```diff
@@ -120,3 +123,28 @@ directory::
 
   $ cd docs
   $ make clean html
+
+PDF Documentation
+^^^^^^^^^^^^^^^^^
+
+To build the PDF version of documentation, `TEX <https://www.tug.org/texlive/quickinstall.html>`_ needs
+to be installed first. Additional requirements of python modules can be met by simply running the following
+commands in docs directory::
+
+  $ pip install -r requirements.txt
```

Comment:
Instead of having a separate file can you follow the pattern in the "HTML Documention" section and just list the command needed to install the requirements inline with the text. Also need to still mention that the TRTIS python package must be installed.

Reply:
Sure. Let me make that change. 

Should TRTIS python package be installed for documentation as well ? Correct me if I am wrong but that doesn't seem to be necessary requirement.

Reply:
TRTIS python package is required so that the API docs get generated correctly for it.